source/Camera/Maintenance/CorrectCameraTrueness/CorrectCameraTrueness.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+This example shows how to perform In-field correction. This involves gathering data from
+a compatible calibration object at several distances, calculating an updated camera
+correction, and optionally saving that new correction to the camera.
+
+The correction will persist on the camera even though the camera is power-cycled or
+connected to a different PC. After saving a correction, it will automatically be used any
+time that camera captures a new point cloud.
+*/
+
+#include <Zivid/Zivid.h>
+
+#include <algorithm>
+#include <iomanip>
+#include <iostream>
+#include <vector>
+
+namespace
+{
+    bool yesNoPrompt(const std::string &question)
+    {
+        while(true)
+        {
+            std::string response;
+            std::cout << question << ": y/n " << std::endl;
+            std::cin >> response;
+
+            if(std::cin.fail() || response == "n" || response == "N")
+            {
+                return false;
+            }
+            if(response == "y" || response == "Y")
+            {
+                return true;
+            }
+            std::cout << "Invalid response. Please respond with either 'y' or 'n'." << std::endl;
+        }
+    }
+
+    std::vector<Zivid::Experimental::Calibration::InfieldCorrectionInput> collectDataset(Zivid::Camera &camera)
+    {
+        std::vector<Zivid::Experimental::Calibration::InfieldCorrectionInput> dataset;
+        std::cout << "Please point the camera at a Zivid in-field calibration object. " << std::endl;
+
+        constexpr char printLine[] = "------------------------------------------------------------------------";
+        while(true)
+        {
+            std::cout << printLine << std::endl;
+            if(yesNoPrompt("Capture (y) or finish (n)?"))
+            {
+                std::cout << "Capturing calibration object" << std::endl;
+                const auto detectionResult = Zivid::Experimental::Calibration::detectFeaturePoints(camera);
+                const auto input = Zivid::Experimental::Calibration::InfieldCorrectionInput{ detectionResult };
```

Comment:
Wouldn't it be better to call this 'InfieldCorrectionInput'?

Reply:
Yeah, good idea. Except with lower-case 'infieldCorrectionInput'.

Reply:
Also in the other sample.